meteor/sofie-server/data-access/repositories/mongo/mongo-entity-converter.ts

```diff
@@ -142,9 +128,9 @@ export class MongoEntityConverter {
 			showStyleVariantId: '', // Todo: figure out where the value for this attribute is
 			studioId: '', // Todo: figure out where the value for this attribute is
 			timing: { expectedDuration: 0, expectedEnd: 0, expectedStart: 0, type: '' }, // Todo: figure out where the value for this attribute is
-			_id: new ObjectId(rundown.id),
+			// _id: new ObjectId(rundown.id),
 			name: rundown.name,
-		}
+		} as unknown as MongoRundown
```

Comment:
Something needs to be cleaned up here.
I belive that we want the conversion of the id and that we don't want to cast it `as unknown`?

Reply:
Removed your temp changes, from our debugging session that i had missed.